Automattic/wp-calypso
# Add loading step on LIB flow

#### Proposed Changes

After the link-in-bio setup step, when submitting the form, the user should see the loading step that is basically a loader while the data is being handled/submitted.

It should look like the [designs](https://www.figma.com/file/BcUdhLlhAp7UHtQHNTI4zA/Tailored-Onboarding-%26-Launchpad?node-id=1865%3A18108) besides the background that is not the one on the design, it will be changed later.

<img width="652" alt="image" src="https://user-images.githubusercontent.com/2653810/183629840-c3919556-579c-41f0-adfa-99c31a614932.png">



#### Testing Instructions

 - Run yarn start from the calypso root folder
 - Navigate to http://calypso.localhost:3000/setup/linkInBioSetup?flow=link-in-bio&siteSlug=YOUR-SITE.wordpress.com
 - Fill the form and click on continue
 - You should see the loader as the described above.
 ( It will be fast given we have little data and the whole action happens quite fast )
- After that you should be redirect to temporary page, `http://calypso.localhost:3000/page/YOUR-SITE.wordpress.com/home`. ( will be changed later )

Related to #66396
Fixes #66396
